# StopEnvironment.srv
string file_name
---
bool success
string message
